src/components/InputField.vue

```diff
@@ -0,0 +1,47 @@
+<template lang="pug">
+  div
```

Comment:
Do you really need this component? does it have any specific logic to create this abstraction?

Reply:
It receives a class from the parent component. Is there a way to avoid defining this element?

Reply:
Not defining it 😃

Reply:
If i dont define a root element, the component doesn't work. 
`Component template should contain exactly one root element. If you are using v-if on multiple elements, use v-else-if to chain them instead.`